Cargo.toml

```diff
@@ -51,6 +51,14 @@ opt-level = 1
 [profile.release]
 panic = "abort"
 
+# Speed up release syncs and sync tests by optimizing all release build code.
+# Some of Zebra's code is CPU-intensive, and needs extra optimizations for peak performance.
+#
+# The `cc` crate automatically applies this setting when compiling C/C++ code:
+# https://docs.rs/cc/latest/cc/struct.Build.html#method.opt_level
+# https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts
+opt-level = 3
```

Comment:
[This is already the default for `release`](https://doc.rust-lang.org/book/ch14-01-release-profiles.html#customizing-builds-with-release-profiles)

Reply:
Hmm, that's weird, I guess the 5 minutes was network variation.

I'll try turning on link-time optimisation instead.